docs/DeveloperGuide.md

```diff
@@ -1,37 +1,176 @@
 # Developer Guide
 
-## Acknowledgements
-
-{list here sources of all reused/adapted ideas, code, documentation, and third-party libraries -- include links to the original source as well}
-
 ## Design & implementation
 
 {Describe the design and implementation of the product. Use UML diagrams and short code snippets where applicable.}
 
+### Results feature
+
+The results feature is facilitated by `ResultsList` and 
+`AnswerTracker`. Both are used to display the results for
+all question sets attempted by the user, including details
+such as specific questions and their respective answer inputs
+from the user.
+
+Given below is an example usage scenario and how the results
+mechanism behaves at each step.
+
+Step 1. The user launches the application for the first time,
+and proceeds to start a game with their chosen topic.
+
+The following sequence diagram shows how the `Results` for
```

Comment:
Hi, the method calling solid line arrow for "new Results()" seems to not be pointing to the start of the activation bar.